tests/test_stats.py

```diff
@@ -53,4 +56,11 @@ def test_mock_pairsam():
     assert stats['dist_freq/1-2/++'] == 1
     assert stats['dist_freq/2-3/++'] == 1
     assert stats['dist_freq/32-56/++'] == 1
+    assert stats['summary/frac_cis'] == 0.6
+    assert stats['summary/frac_cis_1kb+'] == 0
+    assert stats['summary/frac_cis_2kb+'] == 0
+    assert stats['summary/frac_cis_4kb+'] == 0
+    assert stats['summary/frac_cis_10kb+'] == 0
+    assert stats['summary/frac_cis_20kb+'] == 0
+    assert stats['summary/frac_cis_40kb+'] == 0    
```

Comment:
Wow, these are great tests! Is it possible also to test the deduplication stats? 
It's not my idea, though, I found it in https://github.com/open2c/pairtools/issues/5

Reply:
Yeah that's why I didn't notice the recent error in stats with the new dataframe method, because the test file doesn't have an duplicated pairs... SHould be easy to fix